pyproject.toml

```diff
@@ -73,6 +73,7 @@ full = [
     #
     # Until such time that this dependency can be installed through
     # flow.record, we define it as a dependency of dissect.target.
+    "tomli; python_version<'3.11'",
```

Comment:
Can you put this above the comment? The comment applies to the zstandard dependency.